### ISSUE TITLE
AutoComplete Keyboard Navigation Issue during IME Input

### ISSUE THREAD

**Issue by whilemouse** [primefaces/primevue#6644]
### Describe the bug

# AutoComplete Keyboard Navigation Issue during IME Input

##  Issue Description
When using IME input (Korean) the keyboard navigation (arrow keys) in the AutoComplete component doesn't work properly. During IME composition, pressing the down arrow key doesn't move focus to the suggestion list. The arrow keys only work after the IME composition is completed.

### Reproducer

https://stackblitz.com/edit/primevue-4-vite-issue-template-uuqxqx?file=src%2FApp.vue

### PrimeVue version

4.1.1

### Vue version

3.x

### Language

ES6

### Build / Runtime

Vite

### Browser(s)

chrome 130.0.6723.59

### Steps to reproduce the behavior

1. Start typing Korean characters in AutoComplete component
2. When suggestions appear, press the down arrow key
3. Arrow key navigation doesn't work during IME composition

##   Expected Behavior
Arrow key navigation should work even during IME composition to navigate through the suggestion list.
